### ISSUE TITLE
MLOps & Medical Device Regulations compliance

### ISSUE THREAD

**Issue by hshuaib90** [GSTT-CSC/MLOps#1]
Do we need to have a separate repo for Medical Device Regulations compliance? Things like quality management system, requirements traceability etc. Ideally we want a single CI/CD process that is suitable not only for ML but also ML in a regulated environment. Should we do it all in this repo and split it out if it makes sense?

My sense would be let's solve the entire CI/CD pipeline here, and then split out components later.

**Comment by laurencejackson** [comment]
@AnilMistry, copying you in here.

From our discussions yesterday I think maybe the best place us to keep the medical device regulations compliance docs would be in the new project template (which will exist soon). That way whenever a new project is started, the generic compliance docs that are applicable to all projects are readily available there, and we will use rdm for creating project specific docs. 

Does this sound like a good approach?

**Comment by hshuaib90** [comment]
@AnilMistry @laurencejackson what's the progress for this?

**Comment by laurencejackson** [comment]
Me and @AnilMistry have just had a chat about how best to integrate the regulatory docs with the project template. 

We will create a new repo called something like "QMS_protocols' which will be our ground truth tracked version of local protocol documentations. 

The next version of the project_template will include a 'regulations' folder which will contain the rdm generated project-specific documentation and will include a clone of the most recent QMS_protocols repo. by keeping these separate we then know what version of the QMS_protocols the developer was working under at any particular point. When a new version of the QMS_protocols is issued a message will go out to all developers who will pull the most recent version into the copy of the project_template they're working from and then we know that from that point they are working under the newest version of the QMS_protocols. Once this is in place we should be compliant with ISO13485 and IEC62304. 

Does this make sense? Can you think of any other considerations we need to make/ any avoidable inefficiencies here?

**Comment by hshuaib90** [comment]
Thanks, both. Yes that makes sense - can we post the link to the new repo here then we can close this issue. We should also have an empty regulations folder added in the meantime to this repo as a placeholder.

- [ ] Post link of QMS repo to this issue
- [ ] Add placeholder directory to this repo called 'regulations'

**Comment by hshuaib90** [comment]
> Thanks, both. Yes that makes sense - can we post the link to the new repo here then we can close this issue. We should also have an empty regulations folder added in the meantime to this repo as a placeholder.
> 
> * [ ]  Post link of QMS repo to this issue
> * [ ]  Add placeholder directory to this repo called 'regulations'

@AnilMistry these actions are still outstanding, is there anything blocking from completing these two tasks?

**Comment by laurencejackson** [comment]
I'm going to move this issue to the Project_template repo since that is where we're addressing this
